idd/Energy+.idd.in

```diff
@@ -1442,7 +1442,7 @@ RunPeriod,
        \key Yes
        \key No
        \default No
-  A6,  \field Use Weather File Rain Indicators
+  A6,  \field Use Rain Indicators
```

Comment:
Given that I/O Freeze is almost frozen, the IDD change will likely have to wait until after v22.1 release. And it will need doc updates and a mention in the input rule file (because field name changes must be documented for epJSON).

Reply:
I'll remove the I/O change for now.